provider/sweeper_test.go

```diff
@@ -222,3 +226,26 @@ func testSweepActiveActiveSubscriptions(region string) error {
 
 	return nil
 }
+
+func testSweepAcl(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return err
+	}
+
+	rules, err := client.RedisRules.List(context.TODO())
+	if err != nil {
+		return err
+	}
+
+	for _, rule := range rules {
```

Comment:
You should check whether the resource has the `testResourcePrefix` before deleting